src/Types/Output/ListOutputType.hack

```diff
@@ -23,18 +23,26 @@ final class ListOutputType<TInner, TResolved>
         vec<\Graphpinator\Parser\Field\IHasSelectionSet> $parent_nodes,
         GraphQL\ExecutionContext $context,
     ): Awaitable<GraphQL\FieldResult<vec<mixed>>> {
-        $ret = vec[];
-        $errors = vec[];
-        $is_valid = true;
-
-        $results = await Vec\map_async(
+        $results = await Dict\map_async(
```

Comment:
Had to change the result type here to a dict from indices to values so that we can merge it with deferred results later.